src/engine/channelhandle.h

```diff
@@ -12,21 +20,13 @@
 // and equality of ChannelHandles are simple to calculate and a QVarLengthArray
 // can be used to create a fast associative container backed by a simple array
 // (since the keys are numbered [0, num_channels]).
-//
-// A helper class, ChannelHandleFactory, keeps a running count of handles that
-// have been assigned.
 
-#include <QHash>
-#include <QString>
-#include <QVarLengthArray>
-#include <QtDebug>
-#include <memory>
-
-#include "util/assert.h"
-
-// A wrapper around an integer handle. Used to uniquely identify and refer to
-// channels (headphone output, master output, deck 1, microphone 4, etc.) of
-// audio in the engine.
+/// A wrapper around an integer handle. Used to uniquely identify and refer to
+/// channels (headphone output, master output, deck 1, microphone 4, etc.) of
+/// audio in the engine.
```

Comment:
```suggestion
/// audio in the realtime thread without QString comparisons.
```

Reply:
why limit it to any use case? It can be used everywhere.

Reply:
@daschuer I didn't write this, just formatted as documentation ;)

Reply:
But yes, I agree with you